### DISCUSSION TITLE
Out-of-github communication

### DISCUSSION THREAD

**Discussion by ajeckmans** [reqnroll/Reqnroll#140] [Contributor Q&A]
@gasparnagy A while ago you talked about setting up some sort of out-of-github communication, like discord for the smaller discussions, quick questions, etc. Do you have something in place already?

**Comment by gasparnagy** [comment]
@ajeckmans not yet, but I would love to have one. I'm not a big chat person, so I don't know what is trending... I use slack occasionally, but also fine with discord (as long as someone else can take care for setting up the server 😎).

**Reply by ajeckmans** [reply]
I'll look into it then. I don't expect it to be really busy, but it might ease some discussions. I still think most communication should go through GitHub as you have a nice written history in one spot and it does not rely on overlapping active hours, but some discussions are just a bit easier to do first in a more chat-like environment.

**Reply by mbhoek** [reply]
Let me know if I can help, I have maintained a couple of Discord servers.

**Reply by ajeckmans** [reply]
Discord admin is pretty new to me, but it seems fairly straight-forward. You can join here. https://discord.gg/vyZv9z4hGY

**Reply by gasparnagy** [reply]
@ajeckmans (It says expired.) I think the easiest is if @mbhoek sets up a server for us. 

@mbhoek Could you please do it? I think the name should suggest that this is for Reqnoll **contributors**, not general support channel. Do you need any data from us?

**Reply by ajeckmans** [reply]
works for me:
![image](https://github.com/reqnroll/Reqnroll/assets/2088777/ee2f9374-77f9-4e39-81be-39017d5339b5)
The name is taken already, so that would pose a problem

**Reply by mbhoek** [reply]
@gasparnagy The server that @ajeckmans created (and the invite link) works for me, although it is a bit lonely right now 😄 Maybe try this link? https://discord.gg/Qy9Wjm7W

**Reply by gasparnagy** [reply]
I think the server works fine. Could someone please add the invite link to the `CONTRIBUTING.md`?
